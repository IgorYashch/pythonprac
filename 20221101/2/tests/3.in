a = Triangle((0, 0), (-1, -1), (-3, -3))
b = Triangle((1, 1), (1, 2), (4, 2))
c = Triangle((0, 0), (1, 0), (1, 1))

print(abs(a), abs(b))
print(bool(a), bool(b))
print(a < b)
print(a in b)
print(b & c)