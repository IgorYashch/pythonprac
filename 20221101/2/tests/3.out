0.0 1.5
False True
True
True
True
